docker/tap-schema/1015_dp01_dc2_tap_schema.sql

```diff
@@ -0,0 +1,1185 @@
+INSERT INTO tap_schema.schemas11 (schema_name, utype, description, schema_index) VALUES ('dp01_dc2_catalogs', null, 'Data Preview 0 includes five tables based on the DESC’S Data Challenge 2 simulation of 300 square degrees of the wide-fast-deep LSST survey region after 5 years. All tables contain objects detected in coadded images.', 0);
+INSERT INTO tap_schema.tables11 (schema_name, table_name, table_type, utype, description, table_index) VALUES ('dp01_dc2_catalogs', 'dp01_dc2_catalogs.position', 'table', null, 'Select astrometry-related parameters for objects detected in the coadded images, such as coordinates, footprints, patch/tract information, and deblending parameters. (29 columns)', 0);
```

Comment:
Here and in the other `tables11` inserts the table ordering specified by @melissagraham under DM-29735 was not implemented as `table_index` values.  This should be fixed.

Reply:
Indeed the `table_index` values seem not implemented by Felis. 

I may fix it manually, though would need help on what this really means. Do I replace the `table_index` field by 0,1,2,3,4 following the table order CET suggested, and it'd just work?

Reply:
Yes, that works minimally.  We still have a larger issue about how to set `schema_index` so that we get an appropriate order for schemas.

Reply:
Added `table_index` in https://github.com/lsst-sqre/lsst-tap-service/pull/51/commits/aaa8f531780c138085331a914504e242b1011471 for the table order

Reply:
(I'm assuming 0 means higher in the order)

Reply:
Yes, lower-numbered tables (and columns) are displayed first.

I'm not asking for work on `column_index` per @leannep's specification that we not work too hard on optimizing the UX for these "borrowed" tables.  But we are going to want to work on both `column_index` and `principal` for our SDM output in DP0.2.